RecoMuon/TrackerSeedGenerator/plugins/MuonHLTSeedMVAClassifier.cc

```diff
@@ -87,33 +91,58 @@ class MuonHLTSeedMVAClassifier : public edm::stream::EDProducer<> {
                     const reco::RecoChargedCandidateCollection& l2Muons);
 };
 
+bool MuonHLTSeedMVAClassifier::checkMVAFileConsistency(const std::string& weightsFileFullPath,
+                                                       const bool isFromL1) const {
+  tinyxml2::XMLDocument xmlDoc;
+  if (reco::details::hasEnding(weightsFileFullPath, ".xml")) {
+    xmlDoc.LoadFile(weightsFileFullPath.c_str());
+  } else {
+    edm::LogError("MuonHLTSeedMVAClassifier") << "unsupported file extension, it should be a .xml file!";
+  }
+  tinyxml2::XMLElement* root = xmlDoc.FirstChildElement("MethodSetup");
```

Comment:
Maybe `return false` if `root == nullptr` ? (not sure if it's possible that the pointer is null)

Reply:
addressed in the last push: https://github.com/cms-sw/cmssw/pull/45063/commits/fb54b4415e6365dc9395f195fe9cfbf20a8af200